
/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Dataset/SegSperm/train
/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Dataset/SegSperm/test_small
GPU dostÄ™pne: NVIDIA GeForce RTX 3090
Traceback (most recent call last):
  File "/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Kod/train.py", line 415, in <module>
    main(model, train_loader, val_loader, optimizer,scheduler, loss_fn, config.epochs,aug,t_aug,save_name)
  File "/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Kod/train.py", line 299, in main
    plot_sample(vimages, vlbls_oneclass, vpreds_oneclass, ix=1, mode='val')
  File "/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Kod/train.py", line 66, in plot_sample
    has_mask = y[ix].max() > 0
               ~^^^^
IndexError: index 1 is out of bounds for axis 0 with size 1